### ISSUE TITLE
Review for Project 01

### ISSUE THREAD

**Issue by LingyaoMeng** [YingZhang1028/istm-6212#2]
Dear Ying, 

I was able to run through your code smoothly. No fuss. thank you! Love the set up that all the questions were stated cleared with markdown as well as your though process. 

For question 1, part a, we used the exact same code. In part B, instead of "Rom" and "Jul", I used "Rom." and "Jul." to match what appeared in the text. 

For problem 2, part a, we used the exact same code. (soul sister, yo) for part b, I like your answers better because it is more straightforward and only included the requested outcome. I should use csvcut -c2 next time. However, when I tried to run your code, I encountered a slight difference in the outcome. see the attached picture on the right for the difference. I am sure this is some minor error but I think it's worth pointing out. 
![lingyaorun](https://cloud.githubusercontent.com/assets/21350392/18880817/34d9f1e6-84a7-11e6-87c9-6c63765c7477.png)
![originalanswer](https://cloud.githubusercontent.com/assets/21350392/18880818/34e5c980-84a7-11e6-9670-261cb283ab98.png)

For question 3, I like your thought process. In filter_lower.py file, line 12, you used line[:-1] instead of just line. I don't understand that. In filter_split.py file, I have a chunk of code that only allows word that are longer than 2 letters pass instead of passing everything. That's a slight difference. And in filter_stopwords.py, you again used line[:-1]. I wish I can understand why... 

Everything else looks wonderful!

Best Regards, 

Lingyao

**Comment by dchud** [comment]
@LingyaoMeng I agree that the difference in where the write error appeared within the output you showed in the screenshot is minor.  That the error occurs is unfortunate but the results look exactly the same in both, so that's what counts.
